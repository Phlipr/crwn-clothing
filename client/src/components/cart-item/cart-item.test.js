import React from "react";
import { shallow } from "enzyme";

import CartItem from "./cart-item.component";

describe("CartItem component:", () => {
    const mockProps = {
        item: {
            imageUrl: "https://www.google.com",
            name: "Phillip",
            quantity: 2,
            price: 2.50
        }
    };

    let wrapper;

    it("should render the CartItem component", () => {
        wrapper = shallow(<CartItem item={mockProps} />);

        expect(wrapper).toMatchSnapshot();
    });
});