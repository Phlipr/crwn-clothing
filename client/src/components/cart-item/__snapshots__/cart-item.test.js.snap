// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem component: should render the CartItem component 1`] = `ShallowWrapper {}`;
